tutorcodejail/templates/codejail/build/codejail/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM docker.io/ubuntu:20.04 as minimal
+FROM docker.io/ubuntu:22.04 as minimal
```

Comment:
I don't think anything is written in stone, but maybe we should try to keep the same base Ubuntu across images?

Seems like both [discovery](https://github.com/overhangio/tutor-discovery/blob/olive/tutordiscovery/templates/discovery/build/discovery/Dockerfile#L1) and [ecommerce](https://github.com/overhangio/tutor-ecommerce/blob/olive/tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile#L1) are going to keep using 20.04.

Reply:
This was Regis' recommendation

Reply:
But this is already working. Let's not waste the effort 😭